src/README.md

```diff
@@ -0,0 +1,75 @@
+# Use of LabVIEW Component Simulator
+
+The Component Simulator allows you to call a LabVIEW TCP Server to test the TCP communication between the TCP Server and the TCP Client. 
+The LabVIEW TCP Server receives json commands from a Python TCP Client, and send Ack and Events back to the Client, using TCP packets.
+
+## Needed Package
+
+- Caraya 1.0.2.115 (for Unit Testing)
+
+## Use of TCP Server
+
+### Build the Executable
+
+The LabVIEW professional version is needed to build the executable. 
+You can build the executable from within LabVIEW or from the command line, as you prefer:
+
+- From within LabVIEW: select the build all under Build Specifications of tcpip.lvproj. The ComponentSimulator executable will be in the `build/` directory.
+The output log is in the build/buildComponentSimulator.txt.
+- From the command line:
+```sh
+labview64 src/buildApp.vi
+```
+
+### Start Component Simulator
+
+From terminal:
+
+Go to the `build/` directory:
+```sh
+cd  ts_labview_tcp_json/build
+```
```

Comment:
Add the new lines before and after the code block.

Reply:
Done.